oxidation/libparsec/crates/platform_mountpoint/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "libparsec_platform_mountpoint"
+version = "0.0.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[[test]]
+name = "integration"
+path = "tests/mod.rs"
+
+[dependencies]
+libparsec_types = { path = "../types" }
+
+once_cell = "1.17.1"
+regex = "1.7.3"
+
+[dev-dependencies]
+rstest = "0.17.0"
```

Comment:
Not using `parsec_test` ?

Reply:
Not needed for the moment, there is only winify